### DISCUSSION TITLE
Uploading Images in Issue Forms

### DISCUSSION THREAD

**Discussion by arieldiporto95** [community/community#58027] [Projects and Issues]
### Select Topic Area

General

### Select Feature Area

- [X] Issues (The feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).
- [ ] Projects (For planning and tracking work on GitHub - not any open source or other type of project).

### Body

Hello everybody,

for the project of my team we are implementing issue templates, but I can't find a way to let users upload files (such as screenshots) to better understand their problem.

We are thinking about linking an external tool to do it, but we would like to leave it all within Github.

Is there a way to do it?

Thanks in advance.

Ariel

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by aahouzi** [comment]
When using issue templates in GitHub, there isn't a built-in way to directly upload files (such as screenshots) as part of the issue creation process. However, there are a few alternative approaches you can consider to keep everything within GitHub:

1. Encourage users to provide file attachments via GitHub comments: In the issue template, you can instruct users to upload their files to a file-sharing service (such as Dropbox, Google Drive, or GitHub Gist) and provide the link in a comment on the issue. This allows users to share their files without leaving the GitHub platform.

2. Utilize GitHub Actions or integrations: You can leverage GitHub Actions or third-party integrations to automatically capture and attach screenshots or other files to issues. For example, you could set up a workflow using GitHub Actions that triggers on issue creation and performs actions like capturing screenshots and attaching them to the issue.

3. Create a separate issue attachment repository: You can create a separate repository within your organization or user account specifically for users to upload their files. Users can open a new issue in that repository and attach their files there. Then, in the original issue, you can reference the file attachments from the separate repository.

4. Use issue labels or a specific comment format: You can create a specific label or instruct users to follow a specific comment format to indicate that they have attached files elsewhere (e.g., using labels like "With Attachments" or commenting with a specific tag). This way, you can easily identify issues that include attachments and handle them accordingly.

These alternatives provide ways to work within the GitHub platform while allowing users to share file attachments or provide additional information related to their issues. Evaluate which approach best fits your team's workflow and requirements.

Remember to keep an eye on the GitHub platform updates, as new features and capabilities are regularly introduced. There might be future enhancements that directly address the issue of file attachments in issue templates.

I hope this information helps! If you have any further questions, feel free to ask.
